cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -277,10 +277,22 @@ async function fetchAnnotations(frameNumber?: number): Promise<{
         jobInstance, showGroundTruth, groundTruthInstance,
     } = receiveAnnotationsParameters();
 
-    const fetchFrame = typeof frameNumber === 'undefined' ? frame : frameNumber;
-    const states = await jobInstance.annotations.get(fetchFrame, showAllInterpolationTracks, filters);
+    const fetchFrame = typeof predefinedFrame === 'undefined' ? frame : predefinedFrame;
+    let states = await jobInstance.annotations.get(fetchFrame, showAllInterpolationTracks, filters);
     const [minZ, maxZ] = computeZRange(states);
-    if (showGroundTruth && groundTruthInstance) {
+    if (jobInstance.type === JobType.GROUND_TRUTH) {
+        states = states.map((state: ObjectState) => new Proxy(state, {
+            get(_state, prop) {
+                if (prop === 'isGroundTruth') {
+                    // ground truth objects are not considered as gt objects, relatively to a gt jobs
+                    // to avoid extra css styles, or restrictions applied
+                    return false;
+                }
+
+                return Reflect.get(_state, prop);
+            },
+        }));
```

Comment:
Generally Im ok with fix.

What do you think about adding a module to `cvat-ui` containing proxy handlers, maybe somewhere in utils? Usually such handlers are independent objects. It can improve code readability by reducing functions size as we have several such handlers.

Reply:
Is that a common practice or ...? 
Do you have any references?

Reply:
Not sure if thats a common practive or not, but Ive seen a couple of examples where handler object is created somehere outside of the `new Proxy...` in [simple mdn example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy#description) and in [vuejs source code](https://github.com/vuejs/core/blob/0dea7f9a260d93eb6c39aabac8c94c2c9b2042dd/packages/reactivity/src/ref.ts#L287). So I thought maybe we can use that approach to reduce function size. What do you think?

Reply:
I can't see real benefits from this approach.
One more thing is that we use proxy in cvat-core, cvat-canvas, and cvat-ui. So, it turns that we need 3 modules just to store Proxy handlers

Reply:
Ok, lets leave it as it is.